sqflite/lib/src/sentry_database.dart

```diff
@@ -196,4 +196,9 @@ class SentryDatabase extends SentryDatabaseExecutor implements Database {
       }
     });
   }
+
+  @override
+  Future<T> readTransaction<T>(Future<T> Function(Transaction txn) action) {
+    return transaction((txn) => action(txn));
```

Comment:
Also fixed this in #1807. We need to call the internal `_database.readTransaction` for this to be correct, no?